specification/resources/resource-manager/Microsoft.Resources/preview/2024-10-01-preview/deletedResources.json

```diff
@@ -0,0 +1,390 @@
+          "Undelete deleted resource": {
+            "$ref": "./examples/UndeleteDeletedResource.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "responses": {
+          "202": {
+            "description": "Accepted - Returns URL in Location header to query for long-running operation status.",
+            "headers": {
+              "Location": {
+                "type": "string",
+                "description": "URL to get status of this long-running operation."
+              },
+              "Azure-AsyncOperation": {
+                "type": "string",
+                "description": "URL to get status of this long-running operation."
+              },
+              "Retry-After": {
+                "type": "string",
+                "description": "Number of seconds to wait before polling for status."
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DeletedResourcesListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DeletedResourcesDefinition"
+          },
+          "description": "Gets or sets the list of deleted resources."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Gets or sets the URL to get the next set of results."
+        }
+      },
+      "description": "List of deleted resources."
+    },
+    "DeletedResourcesDefinition": {
+      "description": "Deleted Resource information.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/DeletedResourcesProperties"
+        }
+      }
+    },
+    "DeletedResourcesProperties": {
+      "type": "object",
+      "description": "Deleted Resource properties",
+      "properties": {
+        "deletedResourceId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "GUID representing deleted resource id."
+        },
+        "subscriptionId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The Azure subscription ID"
+        },
+        "resourceGroup": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the resource group. The name is case insensitive."
+        },
+        "originalResourceLocation": {
+          "type": "string",
+          "description": "Resource location"
+        },
+        "originalResourceId": {
+          "type": "string",
```

Comment:
>type [](http://example.com/codeflow?start=11&length=4)

samw question about arm-id format. If this is not the case then you should update the description to add what format is expected